### PR TITLE
feat(cards): added `card` component examples

### LINKED ISSUE #417: Add `Cards` component examples
### Subject

`Cards`

### Description

We should add card component examples.

Here are examples to refer to:
- [Article cards](https://ui.mantine.dev/category/article-cards/)
- [Application cards](https://ui.mantine.dev/category/app-cards/)
- [Users](https://ui.mantine.dev/category/users/)
- [Yamada UI Docs](https://yamada-ui.com/examples/cards)

### Are you willing to participate in the development of this feature and create a pull request?

Yes

### PR DESCRIPTION
**Pull request by taroj1205**
Closes #417

## Description

I have added `card` component examples.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: contents/data-display/cards/notifications-card/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "en": {
+    "title": "AccountCard",
+    "description": ""
```
Comment (cidkumagai) [inline contents/data-display/cards/notifications-card/metadata.json:4]: please write in English

File: contents/data-display/cards/notifications-card/metadata.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  },
+  "ja": {
+    "title": "AccountCard",
+    "description": ""
```
Comment (cidkumagai) [inline contents/data-display/cards/notifications-card/metadata.json:8]: please write in Japanese

**Review by cidkumagai — DISMISSED** [review]

File: contents/data-display/cards/notifications-card/metadata.json (hunk 1 above)
Comment (taroj1205) [reply contents/data-display/cards/notifications-card/metadata.json]: I have fixed it, please check.

File: contents/data-display/cards/notifications-card/metadata.json (hunk 2 above)
Comment (taroj1205) [reply contents/data-display/cards/notifications-card/metadata.json]: I have fixed it, please check.

File: contents/data-display/cards/payment-method/index.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    "October",
+    "November",
+    "December",
+  ]
+
+  const currentYear = new Date().getFullYear()
+  const years = Array.from({ length: 10 }, (_, i) => currentYear + i)
+
+  return (
+    <Card maxW="md" rounded="xl" variant="outline" as="form">
```
Comment (illionillion) [inline contents/data-display/cards/payment-method/index.tsx:46]: Please add `preventDefault()`

```suggestion
    <Card maxW="md" rounded="xl" variant="outline" as="form" onSubmit={e => e.preventDefault()} >
```

**Review by illionillion — COMMENTED** [review]
@taroj1205 

Thanks for creating so many samples.
I have made some suggestions and if they are OK, please incorporate them.

File: contents/data-display/cards/account-card/index.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  Heading,
+  Input,
+  Text,
+  VStack,
+} from "@yamada-ui/react"
+import type { FC } from "react"
+
+const AccountCard: FC = () => {
+  return (
+    <Card rounded="xl" variant="outline" w="md" as="form">
```
[the remark this answers is not in the document]
Comment (taroj1205) [reply contents/data-display/cards/account-card/index.tsx:21]: Could you tell me why adding `e.preventDefault` should be added and what's the issue if I don't have it?

File: contents/data-display/cards/bookmark-card/index.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      <CardBody>
+        <Image
+          src="https://images.pexels.com/photos/248195/pexels-photo-248195.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
+          alt="Japan"
+          w="full"
+          rounded="md"
+        />
+      </CardBody>
+
+      <CardFooter>
```
[the remark this answers is not in the document]
Comment (taroj1205) [reply contents/data-display/cards/bookmark-card/index.tsx]: Fixed.

File: contents/data-display/cards/account-card/index.tsx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+          Create an account
+        </Heading>
+
+        <Text color="muted">
+          Enter your email below to create your account.
+        </Text>
+      </CardHeader>
+
+      <CardBody>
+        <ButtonGroup variant="outline" gap="sm" w="full">
```
[the remark this answers is not in the document]
Comment (taroj1205) [reply contents/data-display/cards/account-card/index.tsx]: Fixed.

File: contents/data-display/cards/payment-method/index.tsx (hunk 3 above)
Comment (taroj1205) [reply contents/data-display/cards/payment-method/index.tsx]: Fixed.

File: contents/data-display/cards/account-card/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (taroj1205) [reply contents/data-display/cards/account-card/index.tsx]: I got it, and I have fixed it now.

**Review by illionillion — APPROVED** [review]
LGTM

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**